packages/mongo-nav/src/mongo-select/ProjectSelect.tsx

```diff
@@ -112,6 +120,29 @@ function ProjectSelect({
     }
   };
 
+  const fetchProjectsAsGlobalUser = (searchTerm: string) => {
```

Comment:
Shouldn't `searchTerm` be optional here? You're treating it as though it is in the ternary below.

Reply:
It will always be passed in, but may be an empty string, since it is pulled from `value` and `value` is set using `setState` and should never be nullable. However, assigning a default value to `''` is definitely a good safeguard to add.